File: angular/src/app/_models/shoe-receive/ShoeReceiveDetailDto.ts
export class ShoeReceiveDetailDto {
    ShoeReceiveId : number;
    ShoeId:number;
    OrderQty:number; //sl  đặt
    DeliveryQty:number; //sl giao
    ReceiveQty:number; // sl nhận thực
    ShoeReceivedQtyInStock:number; //sl cần nhận thêm
    CheckReceiveComplete:number; // check xem mã giày đã nhận đủ chưa
    Price:number;
  }

File: AppApi/AppApi.Entities/DTO/Shoe-receive/ShoeReceiveDetailDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppApi.Entities.DTO.Shoe_receive
{
    public class ShoeReceiveDetailDto
    {
        public int ShoeReceiveId { get; set; }
        public int ShoeId { get; set; }
        public int DeliveryQty { get; set; } //sl giao
        public int ReceiveQty { get; set; } // sl nhận thực
        public int ShoeReceivedQtyInStock { get; set; } //sl đã nhận cập nhật vào đặt hàng
        public int CheckReceiveComplete { get; set; }
        public int OrderQty { get; set; } //sl đặt
        public float Price { get; set; }
    }
}
